lib/oper.g

```diff
@@ -2021,6 +2048,8 @@ BIND_GLOBAL( "RECALCULATE_ALL_METHOD_RANKS", function()
             fi;
         od;
     od;
+
+    Assert(2, CHECK_ALL_METHOD_RANKS());
```

Comment:
BTW, with all packages loaded, `CHECK_ALL_METHOD_RANKS` takes about 60 milliseconds, while `RECALCULATE_ALL_METHOD_RANKS` takes 600 ms; so we could just always call `CHECK_ALL_METHOD_RANKS` here.

Also note that `RECALCULATE_ALL_METHOD_RANKS` spends 90% of its time in `RankFilter`, or rather, in `WITH_HIDDEN_IMPS_FLAGS`.

Reply:
I think leaving it in an assert is fine. It does occur to me (not for now) there isn't a way I know of to run Asserts at high level during GAP's loading... which might be useful for testing.